README.md

```diff
@@ -1,13 +1,20 @@
-Go bindings for Apache Mesos
-========
+# Go bindings for Apache Mesos
 
 Very early version of a pure Go language bindings for Apache Mesos. As with other pure implementation, mesos-go uses the HTTP wire protocol to communicate directly with  a running Mesos master and its slave instances. One of the objectives of this project is to provide an idiomatic Go API that makes it super easy to create Mesos frameworks using Go. 
 
 [![Build Status](https://travis-ci.org/mesos/mesos-go.svg)](https://travis-ci.org/mesos/mesos-go) [![GoDoc] (https://godoc.org/github.com/mesos/mesos-go?status.png)](https://godoc.org/github.com/mesos/mesos-go)
 
-## Current Status
-This is a very early version of the project.  Howerver, here is a list of things that works so far:
+## Status
+This project is undergoing a *complete* rewrite happening in the
+[next](https://github.com/mesos/mesos-go/tree/next) branch. These bindings will
+integrate exclusively with the new public Mesos HTTP API.
```

Comment:
That's a good question. We have to understand what the timeline for Mesos to have the executor API out is. In any case, the current version of the bindings would still be available.